File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1073.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

n = parseInt(lines.shift())
        
fim = n
if(n%2!=0){
    fim=n-1
}
    
for(i=2;i<=fim;i+=2){
    console.log(i+"^2 = "+i*i)
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1073.c
#include <stdio.h>

int main(){
    int n, i;
    scanf("%d", &n);
    
    int fim = n;
    if(n%2!=0){
        fim=n-1;
    }

    for(i=2;i<=fim;i+=2){
        printf("%d^2 = %d\n", i, i*i);
    }

    return 0;
}